src/frontend/src/observer/observer_manager.rs

```diff
@@ -116,6 +116,7 @@ impl ObserverState for FrontendObserverNode {
             tables,
             indexes,
             views,
+            subscriptions: _,
```

Comment:
Why ignore `subscriptions` here? Does FE need to cache the subscription catalog? If yes, we should store the `subscriptions` in the FE catalog cache. If not, we should remove `subscriptions` from `MetaSnapshot`